src/gui/src/PreferencesDialog_UI.ui

```diff
@@ -384,45 +384,45 @@
              </property>
             </widget>
            </item>
-		   <item row="4" column="0">
-			 <widget class="QLabel" name="jackBBTSyncLbl">
-			   <property name="minimumSize">
-				 <size>
-				   <width>0</width>
-				   <height>22</height>
-				 </size>
-			   </property>
-			   <property name="text">
-				 <string>BBT sync method</string>
-			   </property>
-			   <property name="toolTip">
-				 <string>Specifies the variable, which has to remain constant in order to guarantee a working synchronization and relocation in the presence of another Jack timebase master.</string>
-			   </property>
-			 </widget>
-		   </item>
-		   <item row="4" column="1">	 
-			 <widget class="QComboBox" name="jackBBTSyncComboBox">
-			   <property name="minimumSize">
-				 <size>
-				   <width>0</width>
-				   <height>22</height>
-				 </size>
-			   </property>
-			   <property name="toolTip">
-				 <string>Specifies the variable, which has to remain constant in order to guarantee a working synchronization and relocation in the presence of another Jack timebase master.</string>
-			   </property>
-			   <item>
-				 <property name="text">
-				   <string>constant measure</string>
-				 </property>
-			   </item>
-			   <item>
-				 <property name="text">
-				   <string>matching bars</string>
-				 </property>
-			   </item>
-			 </widget>
-		   </item>
+           <item row="4" column="0">
+            <widget class="QLabel" name="jackBBTSyncLbl">
+             <property name="minimumSize">
+              <size>
+               <width>0</width>
+               <height>22</height>
+              </size>
+             </property>
+             <property name="toolTip">
+              <string>Specifies the variable, which has to remain constant in order to guarantee a working synchronization and relocation in the presence of another Jack timebase master.</string>
+             </property>
+             <property name="text">
+              <string>BBT sync method</string>
+             </property>
+            </widget>
+           </item>
+           <item row="4" column="1">
+            <widget class="QComboBox" name="jackBBTSyncComboBox">
+             <property name="minimumSize">
+              <size>
+               <width>0</width>
+               <height>22</height>
+              </size>
+             </property>
+             <property name="toolTip">
+              <string>Specifies the variable, which has to remain constant in order to guarantee a working synchronization and relocation in the presence of another Jack timebase master.</string>
```

Comment:
Since it's an acronym, Jack should be written capitalised: JACK

Reply:
out of the subject of this pull req, I don't know if I can do it right here

Reply:
is it ok?

Reply:
Alsa too is an acronym... I would prefer do not changing these names in the current pull request

Reply:
I thought it was a new addition since it's a "+" line.
No worries, it can be done later.
_(but still it can be done here while you're at it, there is not potential complication IMHO)_

Reply:
It appeared as if I had added that lines but it was rewritten by the graphical software. However that file is untouched in the last commits.